tests/acceptance/security/test_athena_sqli.py

```diff
@@ -71,7 +69,7 @@ def test_it_handles_injection_attacks(
         *commenting,
         *new_lines,
     ]:
-        del_queue_items.append(del_queue_factory(i))
+        del_queue_items.append(del_queue_factory(i, str(uuid4())))
```

Comment:
I cleaned up this test a bit even if it wasn't related to this PR. Making all items included with same ID was actually not correctly inserting the matches in dynamo and silently failing.